src/index.js

```diff
@@ -21,6 +21,7 @@ import intlReducer, {
   queryLanguagePack,
   updateLocale,
 } from './intl';
+import { estimatedSize as estimatedServerCacheSize } from './intl/server-cache';
```

Comment:
Can we keep this out of the client API & bundle?

Reply:
sure